FiveCalls/FiveCalls/BlueButton.swift

```diff
@@ -59,10 +59,25 @@ class BlueButton: UIButton {
         }
     }
 
+    override var isEnabled: Bool {
+        didSet {
+            updateColors()
+        }
+    }
+    
     func updateColors() {
         if isHighlighted { backgroundColor = highlightBackgroundColor }
         else if isSelected { backgroundColor = selectedBackgroundColor }
         else { backgroundColor = normalBackgroundColor }
+        
+        if isEnabled {
```

Comment:
Refactored. Also, I found that when I lower the alpha level to make the button appear disabled, that also affects the appearance of the label automatically.

The code to set the alpha level of the label to 1.0 is now used every time updateColors is called, but it's not clear to me why this was needed in isSelected and isHighlighted. I see the same behavior when I remove that line of code.